content/applications/general/email_communication/email_domain.rst

```diff
@@ -152,12 +152,12 @@ also be used for other lesser known providers.
 Use a default email address
 ===========================
 
-To force the email address from which emails are sent, you need to create the following key in
-the System Parameters of the database:
+To force the email address from which emails are sent, you need to use a combination of the following keys in the system parameters of the database:
 
-- If ``mail.default.from`` is set, and contains a full email address, all outgoing emails are sent
-  from the given address. This is a requirement to use `Outlook with Odoo 
-  <https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-microsoft-365-or-office-365#option-1-authenticate-your-device-or-application-directly-with-a-microsoft-365-or-office-365-mailbox-and-send-mail-using-smtp-auth-client-submission>`_.
+- ``mail.default.from``: accepts the local part or a complete email address as value
+- ``mail.default.from_filter``: accepts a domain name or a full email address as value
+
+If the email address of the author does not match `mail.default.from_filter`, the email address is replaced by `mail.default.from` (if it contains a full email address) or a combination of `mail.default.from` and `mail.catchall.domain`.
```

Comment:
``If the email address of the author does not match `mail.default.from_filter`, the 'from' email address is replaced by `mail.default.from` (if it contains a full email address) or a combination of `mail.default.from` and `mail.catchall.domain`. For example "Admin" <admin@example.com> => "Admin" <notifications@mycompany.com>``

``Note: The 'catchall' for the reply-to will remain the same for outgoing emails.''

Reply:
Hey @tiku-odoo  This doc addresses the "how to replace the email from address with a default email address" question for cloud customers using Odoo SMTP. The other doc is for those using a third party SMTP, and requires an explanation of the from filter which should be covered in a separate PR (#1449 is a good start). I don't think we need to go into more detail here about what the mail.defaut.from_filter is, it's a config parameter introduced on cloud databases to mimic the from filter when not using an outgoing email server.

Reply:
@std-odoo Can you check this explanation on the `mail.default.from_filter` for accuracy?

Reply:
@tiku-odoo That's correct :)

Reply:
@jqu-odoo 

Hi Jonathan, 

I hope you're doing well. 

Are you okay with the changes on this PR? If so, when you have a moment can you make the changes on the RST?

I will get it over to MIVU and ZST for a review then. Hope you have a great week :) 

Tim

Reply:
Hey Tim,

I have edited the RST according to the suggested edits, with a few reservations:

- mail.default.from_filter cannot be "empty" (but the key is optional).
- I fear that mentioning catchall here adds noise.

Cheers,

Jonathan